torchtext/csrc/register_bindings.cpp

```diff
@@ -51,12 +51,13 @@ PYBIND11_MODULE(_torchtext, m) {
       .def("__len__", &Vectors::__len__);
 
   py::class_<Vocab>(m, "Vocab")
-      .def(py::init<std::vector<std::string>, std::string>())
+      .def(py::init<std::vector<std::string>>())
       .def_readonly("itos_", &Vocab::itos_)
-      .def_readonly("unk_token_", &Vocab::unk_token_)
       .def("__getitem__", &Vocab::__getitem__)
       .def("__len__", &Vocab::__len__)
       .def("insert_token", &Vocab::insert_token)
+      .def("set_default_index", &Vocab::set_default_index)
+      .def("return_default_index", &Vocab::return_default_index)
```

Comment:
nit: I'd expect "get" instead of "return" as a counterpart for a function that uses "set", which is more in line of the concepts of "getters and setters".